tests/acceptance/helpers.py

```diff
@@ -23,10 +23,8 @@
 
 PROJECT_ROOT = os.path.dirname(os.path.abspath(__file__))
 
-os.environ['OPENAPI_YAML'] = os.path.abspath('api/openapi-spec/openapi.yaml')
-
 spec_dict = None
-with open(os.environ.get('OPENAPI_YAML'), 'r') as stream:
+with open(os.getenv('OPENAPI_YAML_PATH'), 'r') as stream:
```

Comment:
also have a default value in place. So that if user doesn't set a path, the default path can be used instead. Readme specifies all commands as if you are running from the agent root directory. Similarly, set a default yaml path file here from that point of view.